adapter.go

```diff
@@ -110,6 +120,11 @@ func NewAdapter(options ...adapterOption) (persist.Adapter, error) {
 	if err != nil {
 		return nil, err
 	}
+	if a.dbUser != "" {
+		fmt.Println("using db username password as %s, %s", a.dbUser, a.dbPasswd)
```

Comment:
I guess we can live without this line especially that it is logging sensitive data.

Reply:
removed.